src/builder.ts

```diff
@@ -455,6 +455,17 @@ export class SchemaBuilder {
           publisherConfig,
           stage,
         )
+        const mapping = this.dmmf.getMapping(typeName)
```

Comment:
Wonder if `mapping` is the best identifier here, or, is there a way to be more specific.